### ISSUE TITLE
Package Missing - broadwell architecture

### ISSUE THREAD

**Issue by mysteriouss** [SynoCommunity/spksrc#2819]
_For new Package Requests, see the guidelines_  

### Setup
_Package Name:_
_Package Version:_

_NAS Model:_ DS3617xs
_NAS Architecture:_ x86_64
_DSM version:_ 6.1.2

### Expected behavior
Tell us what should happen

Packages like tmux will be seen

### Actual behavior
Tell us what happens instead

Only a few package can be found

### Steps to reproduce
_1._ Open Package Center 
_2._ Add Source : packages.synocommunity.com
_3._ Only a few packages..
<img width="1440" alt="screenshot 2017-07-12 4 54 24" src="https://user-images.githubusercontent.com/985230/28109914-4eaaa07a-6723-11e7-8680-cc96c531ca7e.png">

#### Package log
_Check Package Center or `/usr/local/{package}/var/`_
```
Insert the package log here
```
#### Other logs
_E.g. `/var/log/messages` or `/var/log/synopkg.log`_
```
Insert log here
```

---

```bash
$ curl -v 'http://packages.synocommunity.com'
* Rebuilt URL to: http://packages.synocommunity.com/
*   Trying 104.31.94.179...
* Connected to packages.synocommunity.com (104.31.94.179) port 80 (#0)
> GET / HTTP/1.1
> Host: packages.synocommunity.com
> User-Agent: curl/7.43.0
> Accept: */*
>
< HTTP/1.1 400 BAD REQUEST
< Date: Wed, 12 Jul 2017 08:12:34 GMT
< Content-Type: text/html
< Content-Length: 192
< Connection: keep-alive
< Set-Cookie: __cfduid=dbdfb980f435a119efd5c9af4b8dda4ef1499847154; expires=Thu, 12-Jul-18 08:12:34 GMT; path=/; domain=.synocommunity.com; HttpOnly
< CF-Cache-Status: MISS
< Server: cloudflare-nginx
< CF-RAY: 37d289c975a32e21-NRT
<
<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
<title>400 Bad Request</title>
<h1>Bad Request</h1>
<p>The browser (or proxy) sent a request that this server could not understand.</p>
* Connection #0 to host packages.synocommunity.com left intact
```

```bash
$ dig packages.synocommunity.com

; <<>> DiG 9.8.3-P1 <<>> packages.synocommunity.com
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 55212
;; flags: qr rd ra; QUERY: 1, ANSWER: 2, AUTHORITY: 0, ADDITIONAL: 0

;; QUESTION SECTION:
;packages.synocommunity.com.	IN	A

;; ANSWER SECTION:
packages.synocommunity.com. 93	IN	A	104.31.95.179
packages.synocommunity.com. 93	IN	A	104.31.94.179

;; Query time: 20 msec
;; SERVER: 192.168.3.1#53(192.168.3.1)
;; WHEN: Wed Jul 12 16:13:07 2017
;; MSG SIZE  rcvd: 76
```

```bash
$ curl -v http://104.31.95.179 -i -H 'Host: packages.synocommunity.com'
* Rebuilt URL to: http://104.31.95.179/
*   Trying 104.31.95.179...
* Connected to 104.31.95.179 (104.31.95.179) port 80 (#0)
> GET / HTTP/1.1
> Host: packages.synocommunity.com
> User-Agent: curl/7.43.0
> Accept: */*
>
< HTTP/1.1 400 BAD REQUEST
HTTP/1.1 400 BAD REQUEST
< Date: Wed, 12 Jul 2017 08:13:53 GMT
Date: Wed, 12 Jul 2017 08:13:53 GMT
< Content-Type: text/html
Content-Type: text/html
< Content-Length: 192
Content-Length: 192
< Connection: keep-alive
Connection: keep-alive
< Set-Cookie: __cfduid=d164b1ab3e8f05c3c01623082fc3105d91499847233; expires=Thu, 12-Jul-18 08:13:53 GMT; path=/; domain=.synocommunity.com; HttpOnly
Set-Cookie: __cfduid=d164b1ab3e8f05c3c01623082fc3105d91499847233; expires=Thu, 12-Jul-18 08:13:53 GMT; path=/; domain=.synocommunity.com; HttpOnly
< CF-Cache-Status: MISS
CF-Cache-Status: MISS
< Server: cloudflare-nginx
Server: cloudflare-nginx
< CF-RAY: 37d28bb8e1386eb7-NRT
CF-RAY: 37d28bb8e1386eb7-NRT

<
<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
<title>400 Bad Request</title>
<h1>Bad Request</h1>
<p>The browser (or proxy) sent a request that this server could not understand.</p>
* Connection #0 to host 104.31.95.179 left intact
```

**Comment by KaraokeStu** [comment]
Hi,

The package server does not listen to "normal" http(s) requests and will only listen directly to the Synology Package centre.

You'll need to add https://packages.synocommunity.com to your package centre

**Comment by mysteriouss** [comment]
It's the same, 400 Bad Request.

**Comment by mysteriouss** [comment]
Could you please tell me a path or a tcp port to test the connection? Thanks.

**Comment by KaraokeStu** [comment]
Firstly, please fill out this information:

Setup

Package Name:
Package Version:

NAS Model:
NAS Architecture:
DSM version:

**Comment by mysteriouss** [comment]
Yes, updated. Thanks

**Comment by mysteriouss** [comment]
Ok, beta opened, and the architecture is the reason I think. Thanks..

**Comment by mysteriouss** [comment]
uname -a

Linux DSM6 3.10.102 #15132 SMP Thu Jun 29 07:35:32 CST 2017 x86_64 GNU/Linux synology_broadwell_3617xs

**Comment by KaraokeStu** [comment]
Yup, we're aware of the issue - I'm waiting on the repo owners to sort this out.

**Comment by TalZiv** [comment]
I have managed to compile git, sonarr, mediainfo and transmission for 6.1.
I am struggling to compile python for couchpotato.

Hope this issue will be sorted soon.

**Comment by lennvilardi** [comment]
Could you share the Transmission package for broadwell ?

**Comment by ymartin59** [comment]
Here it is: 
[transmission_broadwell-6.1_2.92-12.spk.zip](https://github.com/SynoCommunity/spksrc/files/1327305/transmission_broadwell-6.1_2.92-12.spk.zip)

**Comment by dmessing3112** [comment]
Hi i was wondering if you anyone could post the spk for GIT and python 2.7 for broadwell if anyone has managed to get it working.
I am trying to install sickrage and couchpotato via dsm 6.x and no luck.
I have had some success using docker for couchpotato but not for sickrage.

**Comment by ymartin59** [comment]
@m4tt075 May you please help and produce these packages ?

**Comment by m4tt075** [comment]
Here is git (compiled straight from master):
[git_broadwell-6.1_2.11.0-10.spk.zip](https://github.com/SynoCommunity/spksrc/files/1435815/git_broadwell-6.1_2.11.0-10.spk.zip)

Python does not compile though. @ymartin59 There are several issues. I will create a WIP PR.

**Comment by Safihre** [comment]
Test packages for DSM6.1 and various architectures are available here #3138. They also include a new way of handling permissions.
We hope to publish the packages to the official repo soon and appreciate all testing and reports on the functioning of these packages.
